libavfilter/signature_lookup.c

```diff
@@ -199,7 +199,8 @@ static MatchingInfo* get_matching_parameters(AVFilterContext *ctx, SignatureCont
 
     /* initialize houghspace */
     for (i = 0; i < MAX_FRAMERATE; i++) {
-        hspace[i] = av_malloc_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem));
+        hspace[i] = av_malloc_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem)); 
+        memset(hspace[i], 0x00, sizeof(hspace_elem) * (2 * HOUGH_MAX_OFFSET + 1));
```

Comment:
```suggestion
        hspace[i] = av_mallocz_array(2 * HOUGH_MAX_OFFSET + 1, sizeof(hspace_elem)); 
```
Cleaner way to do this is using `av_mallocz_array` which initializes all elements with `0x00`